openslides/core/static/js/core/site.js

```diff
@@ -355,10 +355,32 @@ angular.module('OpenSlidesApp.core.site', [
                                     alreadyConverted.push(stack);
                                     break;
                                 case "img":
+                                    // TODO: need a proper way to calculate the space
+                                    // left on the page.
+                                    // This requires further information
+                                    // A4 in 72dpi: 595px x 842px
+                                    var maxResolution = {
+                                        width: 590,
+                                        height: 840
+                                    },
+                                        width = parseInt(element.getAttribute("width")),
```

Comment:
you seem to be confused by the missing var. This conding style is consistent with the surounding coding stile in the same file.

```
var maxResolution = {
    width: 590,
    height: 840
},
    width = parseInt(element.getAttribute("width")),
    height = parseInt(element.getAttribute("height"));
```
is equivalent to:

```
var maxResolution = {
    width: 590,
    height: 840
};
var width = parseInt(element.getAttribute("width"));
var height = parseInt(element.getAttribute("height"));
```